### DISCUSSION TITLE
Redesign Tailor Dashboard

### DISCUSSION THREAD

**Discussion by Siddhant-Patil0203** [UBA-GCOEN/StichHub#223] [Ideas]
Currently, the Tailor Dashboard looks like this :
Desktop View : 
![image](https://github.com/UBA-GCOEN/StichHub/assets/77800620/b1786efb-e38d-4fe7-b860-d6cd5818948a)

Mobile View :
![image](https://github.com/UBA-GCOEN/StichHub/assets/77800620/119b6bf2-6e18-4c1a-aab1-3f1ff3d5fd81)

## Any Suggestions 🤔💭, How to make it attractive and beautiful and Improve its UI / UX

**Comment by vk20july** [comment]
@Siddhant-Patil0203 may be we can add some trending clothes nowdays and add some information of ```what you get as a tailor```
also we can add some animation like this https://www.behance.net/gallery/92818519/Ticket-Tailor
but as it is dashboard so we have to add only 1-2 thing

**Reply by Siddhant-Patil0203** [reply]
we can add this to a landing page, how will we add this in the dashboard ??

**Reply by vk20july** [reply]
by going into dashboard file and add some html,css and js code 
correct me if i am wrong or can you tell me sir why this is not possible?

**Reply by Siddhant-Patil0203** [reply]
I mean, there is no necessity to add this in the dashboard, because it will make the dashboard more congested

**Reply by vk20july** [reply]
ok sir

**Comment by rahulaarora** [comment]
we can add some borders and also we can use glassmorphism for cool glass effect like for cards, etc

**Reply by Siddhant-Patil0203** [reply]
Sound great !! We can do this
